src/profilers.cpp

```diff
@@ -325,11 +336,15 @@ namespace aperf {
           break;
 
         case ERROR_CONVERSION_TO_FLOAT:
-          print(hint + "metric reader tried to parse float from metric command", true, true);
+          print(hint + "metric reader failed to parse float from metric command", true, true);
           break;
 
         case ERROR_PARSING_CONNECTION_INSTRS:
-          print(hint + "metric reader tried to parse instructions to connect to server", true, true);
+          print(hint + "metric reader failed to parse instructions to connect to server", true, true);
+          break;
+
+        case ERROR_USER_REGEX_MATCH:
+          print(hint + "input regex did not match any metric command output ", true, true);
```

Comment:
You should finish your error messages with a dot :)

Reply:
Done :+1: